### PR TITLE
Configuretransactions

### PR DESCRIPTION
**Pull request by EthanAuyeung**
# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Created transaction queries to add, get all, add voucher, and remove voucher

## Any bugs you encountered or still having trouble with? 🐛
Haven't been able to test the add voucher and remove voucher due to bugs with my expo. If someone could test for me it would be great. I already included the test function

## Resources 📔
_link online resources and related PRs_ 

🥦 CC: @oahnh

### CODE REVIEW

File: src/database/queries.ts [hunk 1]
```diff
@@ -100,3 +103,65 @@ export const getVouchersByVendorUuid = async (
     throw e;
   }
 };
+
+/**
+ * Get all transactions from the `transactions` collection
+ */
+export const getAllTransactions = async (): Promise<Transaction[]> => {
```
Comment (allisonhongberkeley) [inline src/database/queries.ts:173]: getAllTransactions() and getTransaction() don't work and return empty arrays. See above comment for how to fix.

code to test queries in console: 

const testQueries = async () => {
     getTransaction('8HEjHtFLvaMKsyEtLd0g').then(value => {
     console.log(value);
     });
};

testQueries();

File: src/types/types.tsx [hunk 2]
```diff
@@ -19,3 +19,10 @@ export enum VoucherType {
   ORANGE,
   PURPLE,
 }
+
+export type Transaction = {
+  date: string;
+  uuid: uuid;
+  vendorUUID: uuid;
+  vouchers: Array<uuid>;
```
Comment (allisonhongberkeley) [inline src/types/types.tsx:40]: Not sure line 27 is accurate. Is the type of **vouchers** supposed to be an Array of uuid's?

File: src/database/queries.ts [hunk 3]
```diff
@@ -5,19 +5,22 @@ import {
...
 } from 'firebase/firestore';
-import { uuid, Vendor, Voucher } from '../types/types';
+import { uuid, Vendor, Voucher, Transaction } from '../types/types';
 import fbApp from './clientApp';
 
 const db = getFirestore(fbApp);
 const testColl = collection(db, 'test-col');
 const vendorCollection = collection(db, 'vendors');
 const voucherCollection = collection(db, 'vouchers');
+const transactionCollection = collection(db, 'transaction');
```
Comment (allisonhongberkeley) [inline src/database/queries.ts]: 'transaction' should be 'transactions' -- this fix should make the queries work.

File: src/database/queries.ts [hunk 4]
```diff
@@ -145,3 +153,109 @@ export const getVouchersByVendorUuid = async (
...
+    console.warn('(getTransaction)', e);
+    throw e;
+  }
+};
+
+/**
+ * Add a Voucher to the voucher array in the `Transaction` collection.
+ */
+//TODO: configure add + removing vouchers from transaction
+export const addVoucherToTransaction = async (
```
Comment (oahnh) [inline src/database/queries.ts:233]: @EthanAuyeung implement the following two queries

**Pull request closed without merging**